vnext/Microsoft.ReactNative/ReactHost/ReactInstanceWin.cpp

```diff
@@ -38,9 +38,9 @@
 
 #include <codecvt>
 #include <locale>
-#else
-#include "ChakraRuntimeHolder.h"
 #endif
+#include "ChakraRuntimeHolder.h"
```

Comment:
>ChakraRuntimeHolder [](start = 10, length = 19)

Do we have to use it even for V8?

Reply:
Yes, this change enables runtime selection of JSI engine.

---
In reply to: [387947999](https://github.com/microsoft/react-native-windows/pull/4234#discussion_r387947999) [](ancestors = 387947999)